core/arch/arm/plat-imx/drivers/tzc380.c

```diff
@@ -2,36 +2,106 @@
 /*
  * Copyright 2019 Pengutronix
  * All rights reserved.
+ * Copyright 2019 NXP
  *
  * Rouven Czerwinski <entwicklung@pengutronix.de>
  */
 
 #include <drivers/tzc380.h>
 #include <imx-regs.h>
 #include <imx.h>
+#include <kernel/panic.h>
 #include <mm/core_memprot.h>
 #include <mm/generic_ram_layout.h>
 
+#if (defined(PLATFORM_FLAVOR_mx6solosabresd) \
+	|| defined(PLATFORM_FLAVOR_mx6solosabreauto) \
+	|| defined(PLATFORM_FLAVOR_mx6sxsabresd) \
+	|| defined(PLATFORM_FLAVOR_mx6sxsabreauto) \
+	|| defined(PLATFORM_FLAVOR_mx6ulevk) \
+	|| defined(PLATFORM_FLAVOR_mx6ul9x9evk) \
+	|| defined(PLATFORM_FLAVOR_mx6ullevk) \
+	|| defined(PLATFORM_FLAVOR_mx7dsabresd))
 void imx_configure_tzasc(void)
 {
+	uint8_t region = 1;
+	vaddr_t addr;
```

Comment:
minor: default init from line 28, i.e. `vaddr_t addr = core_mmu_get_va(TZASC_BASE, MEM_AREA_IO_SEC);`

Reply:
Addressed in a6bca6b924766606e377fc187444f92ce92122dc